spanner/README.rst

```diff
@@ -12,3 +12,132 @@ Quick Start
 .. code-block:: console
 
     $ pip install --upgrade google-cloud-spanner
+
+
+Authentication
+--------------
+
+With ``google-cloud-python`` we try to make authentication as painless as
+possible. Check out the `Authentication section`_ in our documentation to
+learn more. You may also find the `authentication document`_ shared by all
+the ``google-cloud-*`` libraries to be helpful.
+
+.. _Authentication section: https://google-cloud-python.readthedocs.io/en/latest/core/auth.html
+.. _authentication document: https://github.com/GoogleCloudPlatform/gcloud-common/tree/master/authentication
+
+
+Using the API
+-------------
+
+Cloud Spanner is the world’s first fully managed relational database service
+to offer both strong consistency and horizontal scalability for
+mission-critical online transaction processing (OLTP) applications. With Cloud
+Spanner you enjoy all the traditional benefits of a relational database; but
+unlike any other relational database service, Cloud Spanner scales
+horizontally to hundreds or thousands of servers to handle the biggest
+transactional workloads. (`About Cloud Spanner`_)
+
+.. _About Cloud Spanner: https://cloud.google.com/spanner/
+
+
+Executing Arbitrary SQL in a Transaction
```

Comment:
I am wondering if we need so many examples here. We already have pretty comprehensive set of examples in getting started guide. We can just have a simple quick start which demonstrates how to create a database object and then maybe make a simple sql query using that. We can point to getting started for more examples.